### PR TITLE
Update Button padding.

### PR DESCRIPTION
**Pull request by ImogenF**
### Summary

<blockquote class="trello-card"><a href="https://trello.com/c/BVsaoPw9/282-button-horizontal-padding">Button horizontal padding</a></blockquote>

### To be tested

Tester 1 (Tejas)
- [x] No error/warning in the console
- [x] The netlify build is working

Tester 2 (Peter)

- [x] No error/warning in the console
- [x] The netlify build is working

### CODE REVIEW

**Comment by ImogenF** [comment]
[Button horizontal padding](https://trello.com/c/BVsaoPw9/282-button-horizontal-padding)

File: packages/components/src/utils/constants.ts [hunk 1]
```diff
@@ -145,6 +145,9 @@ const space = {
   /** Small space is `8px` */
   small: 8,
 
+  /** Medium space is `12px` */
+  medium: 12,
```
Comment (TejasQ) [inline packages/components/src/utils/constants.ts]: 👍

**Comment by TejasQ** [comment]
I think it's an _"either or"_ case. 

I would love to see all the buttons as disabled and as condensed. If we think it's bloated, I'd call into question why we even need the second row of disabled styles.

I'm fine with:
a) having _one_ disabled button for show, and _one_ condensed, or
b) having all buttons with all props and a lot of duplication displayed as examples.

I think it strange to have all normal, and disabled states displayed as they are now, but only one condensed example.

**Comment by TejasQ** [comment]
What do you prefer? @ImogenF @peterszerzo @fabien0102?

**Comment by ImogenF** [comment]
My thought process was that disabled looks different depending on the colour of the button. Condensed buttons do not change the color in any way, just the size, so showing condensed buttons for every possible color seems redundant.

**Comment by peterszerzo** [comment]
Nothing wrong with a little redundancy with a huge matrix of all kinds of shapes and sizes I'd say - if anything, it can help with design decisions layout out all options one can choose from when working out how a new UI element should look like.

**Comment by ImogenF** [comment]
How about I get rid of the condensed button in the colours row, and add to the next row down?

image

**Comment by TejasQ** [comment]
That makes a lot of sense to me. 👍

**Comment by peterszerzo** [comment]
Tells the whole story I'd say :).

**Comment by TejasQ** [comment]
@micha-f how does it look?

**Comment by micha-f** [comment]
Looks great 👍

**Pull request merged**